README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are empowered to call out and attempt to resolve disrespectful behavior, and admins are empowered to enforce the Code of Conduct.
+
+## The Short Version
+
+Be respectful of other people; respectfully call out anyone behaving inappropriately, ask them to stop, and direct them to this Code of Conduct; and if you need help resolving an issue, contact the admins. If you are unable to follow our Community Code of Conduct you will be removed from HangOps.
+
+## The Long Version
+
+### Respect
+
+HangOps is a community of people who care about running production systems reliably. We have diverse backgrounds, skills, talents, experiences, cultures, and opinions. We are an inclusive environment and we treat all individuals respectfully.
+
+We value respectful behavior above individual opinions.
+
+Respectful behavior includes:
+
+* Be considerate, kind, constructive, and helpful.
+* Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+* If you’re not sure, ask someone instead of assuming. No, really. Just ask the admins.
+* Model respectful behaviour, but also address disrespect when you see it. One of the best ways to create a respectful community is to speak up when you see disrespectful behavior—and not just rely on admins to act!
+
+### Resolve Peacefully
+
+If you see someone behaving disrespectfully, we urge you to respectfully dissuade them from such behavior.
```

Comment:
If observation is a required step, I want to point out that it is often trivial for abuse to occur without being observed or noticed by a non-victim.

This kind of language allows abusers to wiggle-out of their individual responsibilities to be respectful. Trolling and gaslighting often rely explicitly on the fact that non-targets (3rd parties) may observe the behavior and not become alarmed.

For example, If an abuser follows a victim into hangops, and their presence is simply enough to cause anxiety for their victim. This anxiety is not observed by anyone, nor is it easy for the victim to explain that the abuser being present is an act of abuse. 

The wording here weakens the responsibility, accountability, and enforcement code of conduct. Can the community and admins protect victims and the community if we require behavioral observation and the enforcement mechanism is a weak-sounding "respectfully dissuade"?

Without stronger wording, I wonder if a victim would choose to "respectfully resolve" by removing themselves from our community. Further, this victim-self-removal allows the abusive behavior to remain unchecked by the community, which weakens our community. I personally do not want this outcome and think that the focus on respectful, peaceful resolution is guiding towards it.